src/popup/modules/QrLib/kjua.js

```diff
@@ -3,12 +3,12 @@
  *
  * @module QrLib/kjua
  * @requires /common/modules/Logger
- * @requires qrerr
+ * @requires QrErrors
  */
 /* globals kjua */
 
 import * as Logger from "/common/modules/Logger/Logger.js";
-import * as QrErr from "./qrerr.js";
+import * as QrError from "./QrErrors.js";
```

Comment:
Oh, this can easily result in typos. So better either use signular or plural form here... 😄